keyvault/client/src/main/java/com/azure/keyvault/KeyVaultErrorCodeStrings.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.keyvault;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+
+class KeyVaultErrorCodeStrings {
```

Comment:
Not a big fan of the current structure of this class. The methods inconsistently call `loadProperties()`, there are no default strings (in case the properties file can't be found), and it conflates loading the properties with returning them (meaning each new string requires a new method). I would consider simply exposing the strings (as they are now) and remove the getter methods. People can then pass the appropriate property string into a general getter method. I see no value in having `_PROPERTY` as a suffix. Either it needs to be a prefix (to aid in discovering all such properties), or to remove it entirely (which is the right answer here).

Reply:
And move any relevant comment text to the static field, rather than the method that you should be removing.

Reply:
Updated.